t/page/needed.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../lib/page/needed'
+require 'minitest/autorun'
+require 'pry'
+require 'vcr'
+
+describe 'Needed' do
+  VCR.configure do |config|
+    config.allow_http_connections_when_no_cassette = true
+    config.cassette_library_dir = 't/fixtures/vcr_cassettes'
+    config.hook_into :webmock
+  end
+
+  subject do
+    Page::Needed.new(access_token: ENV['GITHUB_ACCESS_TOKEN'])
```

Comment:
You're still examining the environment in the test, which means these tests fail when I run them, as I don't have a GITHUB_ACCESS_TOKEN set. 

You also don't seem to have a test for the case where someone doesn't have it set, making sure that it throws the correct Error.